app/config.js

```diff
@@ -201,7 +201,7 @@ tag.listView()
         }),
         nga.field('custom', 'template')
             .label('Upper name')
-            .template('{{ entry.values.name.toUpperCase() }}')
```

Comment:
`{{ entry }}` is Angular syntax, not React one (`{entry}`). So, we can't strictly duplicate configuration between two admins.

I'm working on string compilation for React.